addons/osio-addon/src/main/java/io/fabric8/forge/generator/kubernetes/CreateBuildConfigStep.java

```diff
@@ -359,16 +359,18 @@ public Result execute(UIExecutionContext context) {
 
         if (addCI) {
             String discoveryNamespace = kubernetesClientHelper.getDiscoveryNamespace(jenkinsNamespace);
-            String jenkinsUrl = null;
-            try {
-                jenkinsUrl = KubernetesHelper.getServiceURL(kubernetesClientHelper.getKubernetesClient(), ServiceNames.JENKINS, jenkinsNamespace, "https", true);
-                discoveryNamespace = jenkinsNamespace;
-            } catch (Exception e) {
-                if (!discoveryNamespace.equals(jenkinsNamespace)) {
-                    try {
-                        jenkinsUrl = KubernetesHelper.getServiceURL(kubernetesClientHelper.getKubernetesClient(), ServiceNames.JENKINS, discoveryNamespace, "https", true);
-                    } catch (Exception e2) {
-                        throw new BadTenantException("Failed to find Jenkins URL in namespaces " + discoveryNamespace + " and " + jenkinsNamespace + ": " + e, e);
+            String jenkinsUrl = System.getenv("JENKINS_URL");
```

Comment:
So that I understand this correctly, this _jenkinsUrl_ will later then become the base URL for the webhook? A new user gets provisioned, logs in, steps through the codebase creation wizard, selects "enable webhooks" and then will end up with a webhook in the repo of the form: https://\<jenkinsUrl>/github-webhook/?

Reply:
Would JENKINS_PROXY_URL not be a better name? @vpavlin WDYT?

Reply:
Can we get this into pro-preview? Or is it already there?

Reply:
> will end up with a webhook in the repo of the form:

Correct

I'd go with `JENKINS_URL` - if you think about another potential case - Launcher could be deployed in environment where there is one common Jenkins for all projects - ours is a special case where the Jenkins is not actually Jenkins but a proxy going to a random Jenkins.

Reply:
About to push new config map to prod-preview

Reply:
> I'd go with JENKINS_URL

Ok